shared/utils/graphql.ts

```diff
@@ -0,0 +1,273 @@
+import type {
+  AssertionGraphEntity,
+  ChainId,
+  ChainName,
+  OOV2GraphEntity,
+  OracleType,
+  ParsedOOV1GraphEntity,
+  ParsedOOV2GraphEntity,
+  PriceRequestGraphEntity,
+  RequestState,
+} from "@shared/types";
+import { BigNumber } from "ethers";
+
+export function handleGraphqlNullableStringOrBytes<
```

Comment:
these functions take in their values in the form `{key: value}` so we can easily know what key we are dealing with and show a warning about that specific key if the type passed in is wrong (this is often a source of silent bugs)